packages/react/package.json

```diff
@@ -1,74 +1,32 @@
 {
-  "name": "app_scaffold",
-  "version": "2.0.0",
-  "description": "A scaffold for developers to build ZAF v2 apps",
-  "keywords": [
-    "zendesk",
-    "app",
-    "framework"
-  ],
-  "author": "Zendesk",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": ">=18.12.1"
-  },
+  "name": "react_js-template",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
   "scripts": {
-    "test": "jest",
-    "watch": "webpack --watch --mode development",
-    "build:dev": "webpack --mode development",
-    "build": "webpack --mode production",
-    "lint": "standard",
-    "start": "zcli apps:server dist"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "^7.16.5",
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.20.2",
-    "@babel/preset-react": "^7.18.6",
-    "babel-jest": "^29.3.1",
-    "babel-loader": "^9.1.0",
-    "clean-webpack-plugin": "^4.0.0",
-    "copy-webpack-plugin": "^11.0.0",
-    "css-loader": "^6.7.3",
-    "html-webpack-plugin": "^5.5.0",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "@testing-library/react": "^12.1.5",
-    "mini-css-extract-plugin": "^2.7.2",
-    "postcss": "^8.4.20",
-    "postcss-import": "^15.1.0",
-    "postcss-loader": "^7.0.2",
-    "postcss-preset-env": "^7.8.3",
-    "standard": "^17.0.0",
-    "webpack": "^5.75.0",
-    "webpack-cli": "^5.0.1"
+    "dev": "vite --port 3000",
+    "build": "vite build --mode production",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier --write .",
+    "test": "vitest"
```

Comment:
Mind if we match the previous commands? I suspect there's some documentation for it in various places. Probably easier to just keep it the same than track it down and change it :)

Reply:
I will review the preview ones and match them to be the same, just keep in mind we have two stages one for development and one for production (Once we are ready to submit the production version), we might have one extra comand.